shared/scss/_tracker-network-icons.scss

```diff
@@ -1,5 +1,6 @@
 /* Tracker Network Icons */
-$icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "casale", "chartbeat", "comscore", "criteo", "facebook", "fox", "generic", "google", "mixpanel", "moat", "newrelic", "nielsen", "openx", "outbrain", "quantcast", "salesforce", "taboola", "twitter", "yahoo", "yandex");
+$icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "chartbeat", "comscore", "facebook", "fox", "generic", "google", "mixpanel", "moat", "nielsen", "outbrain", "quantcast", "taboola", "twitter", "yahoo", "yandex");
+$grey-icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "casale", "chartbeat", "comscore", "criteo", "facebook", "fox", "google", "mixpanel", "moat", "newrelic", "nielsen", "openx", "outbrain", "quantcast", "salesforce", "taboola", "twitter", "yahoo", "yandex");
```

Comment:
Different list since we have different sets for the pill version and for the subview.

Reply:
Can you give these a variable that is descriptive of its function vs. appearance?  Perhaps `$subview-icons` or something like that.

Reply:
Also, what does the `-g-` in the middle of the .svg file names indicate? i.e. `shared/img/tn-g-twitter.svg` Can this also be more descriptive? Nobody is going to know what -g- means.

Reply:
g is for grey, but yeah you're right, maybe tn-subview-*.svg would work better